tests/test_github.py

```diff
@@ -202,6 +202,36 @@ def test_github_anno_prs(self) -> None:
             """,
         )
 
+        # Test anno-prs using custom remote URL provided by git config keys
+        (
+            self.repo_sandbox.add_git_config_key('machete.github.remote', 'custom_origin')
```

Comment:
For clarity, will the test still pass if these `add_git_config_key` were not present? 🤔

Reply:
Yes, it's the test case just above this one

Reply:
Hmmm then IIUC the test doesn't really serve its purpose... the entire `machete.github.*` feature could be **un**implemented and the test will still pass! ☹️

Reply:
Pls make sure that the tests is written in such a way the setting of `machete.github.*` config keys is a necessary condition for it to pass

Reply:
@PawelLipski Hmm, i am not sure that we are on the same page here, to summarise: **first** part of the test case is as before, checks if it works without setting the keys since they are not mandatory, **second** part of the test tests the same scenario as the first one but this time using the newly added config keys, wdyt?

Reply:
So  the **second** part of the test case will only pass if the keys are present

Reply:
Oh wait, i think i understand what you mean, the **second** part of the test case will pass even when the keys are not present, which is pointless, i'll fix it asap